browser-test/src/support/admin_programs.ts

```diff
@@ -125,6 +128,28 @@ export class AdminPrograms {
     ).not.toContain(statusString)
   }
 
+  async expectApplicationHasStatusStringForBulkStatus(
```

Comment:
also not gonna block cause I know you're trying to keep it up to date, but naming this with the flag, like `expectApplicationHasStatusStringWithBulkStatusEnabled()` is slightly clearer

Reply:
the reason I xxxForBulkStatus() because it will be easier to delete in the future. They will turn it on very soon.